### ISSUE TITLE
please package an appimage of Ring

### ISSUE THREAD

**Issue by star-buck** [appimage-packages/ring#1]
The reason is that current deb packages have some weird / old deps, so an appimage would be great:
https://tuleap.ring.cx/plugins/mediawiki/wiki/ring/index.php/Build_Instructions

Homepage:
https://ring.cx/

PS:
Please add a link to the latest appimage for download in each repo field next to Description:
![snap 2017-03-08 at 12 20 30](https://cloud.githubusercontent.com/assets/840070/23702000/a8fa84ec-03f9-11e7-9ff5-eae9331267a9.png)

**Comment by ScarlettGatelyMoore** [comment]
ok will do

**Comment by star-buck** [comment]
thanks :) also you could add all done appimages in the "Website" field, so its easy to see the latest link

**Comment by star-buck** [comment]
btw. if you happen to find any weblink that shows you the code of the main stable branch, please post it here.

**Comment by ScarlettGatelyMoore** [comment]
https://github.com/savoirfairelinux/ring-project/tree/production ?

**Comment by star-buck** [comment]
Awesome, thanks!

**Comment by ScarlettGatelyMoore** [comment]
GCC in trusty is too old, rebuilding docker image with newer gcc, but this will rebuild everything so it will be awhile..

**Comment by star-buck** [comment]
Thanks. Any ETA for "a while"?

**Comment by ScarlettGatelyMoore** [comment]
still building! QT and Qtwebengine can take 6+ hours. I have nothing planned this weekend so will work on it.

**Comment by star-buck** [comment]
did it build?

**Comment by ScarlettGatelyMoore** [comment]
yes. now working out other missing deps in ring client, namely ffmpeg bits.

**Comment by ScarlettGatelyMoore** [comment]
@star-buck this is tested as far as it runs on my pc.
Functionality in usage needs testing. I am not familiar enough with this software.

https://s3-eu-central-1.amazonaws.com/ds9-apps/ring-master-appimage/ring-latest-x86_64.AppImage

Thanks!
